.husky/post-merge

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 . "$(dirname "$0")/_/husky.sh"
 
-yarn install --frozen-lockfile
```

Comment:
I did this so it would be consistent with the lock file, forcing a change if something was out of date.  did that break?

Reply:
Reverted, had to install yarn. :') wasn't in my typical workflow; I usually default to just NPM.